servers/sanat/site.yml

```diff
@@ -7,6 +7,8 @@
 - hosts: web
   become: yes
   roles:
+    - role: kielipankki.common.backup
```

Comment:
Should this be listed as dependency?

Reply:
It is part of this: https://github.com/CSCfi/Kielipankki-palvelut/blob/cc3161067239c95d041a87d3c521f39e973f78d7/servers/sanat/requirements-pouta_creation.yml#L4C11-L4C62

Reply:
Okay.